include/picongpu/simulation_defines/param/laser.param

```diff
@@ -369,6 +369,92 @@ namespace picongpu
         constexpr float_X LASER_PHASE = 0.0;
     }
 
+    namespace laserExpRampWithPrepulse
+    {
+    // Intensities of prepulse and ASE
```

Comment:
this is still not ASE.

here is the interface the user faces when setting up the new laser profile.
you need to explain
- units and
- meaning / influence

and each variable